flutter-intellij-community.iml

```diff
@@ -42,7 +42,7 @@
       </library>
     </orderEntry>
     <orderEntry type="library" name="snakeyaml" level="project" />
-    <orderEntry type="library" name="Dart SDK" level="project" />
-    <orderEntry type="library" name="Dart Packages" level="project" />
+    <orderEntry type="library" exported="" name="Dart SDK" level="project" />
```

Comment:
Is the `exported=""` necessary here? likely harmless w/ or w/o it?

Reply:
No idea. I didn't edit this file by hand.